### ISSUE TITLE
[Accessibility] Illogical reading order for the popover content in status bar.

### ISSUE THREAD

**Issue by Krituhar** [microsoft/vscode#227565]
- VS Code Version: 1.88.0
- OS Version: MacOS Sonoma (v14.6.1) 

**[Issue]**
The popover content available over the braces control does not have proper reading order. The popover content is placed after the status bar in the DOM structure.

**[User Impact]**
Screen reader users will be unable to understand this content.

**[Code Reference]**
```
<div class="monaco-hover-content" style="overflow: hidden;"><div class="hover-row markdown-hover">
…
<span class="label">Partial Mode</span>
…
</div>
```
**Steps to Reproduce:**
1. Open VScode and hover over the {} JavaScript tab on the bottom of status bar, you can see the popover content, where the reading order is illogical. Ensure that the reading order of content is logical. Place the popover content before/within the status bar's "{}JavaScript" tab in the DOM structure.

Screenshot of issue:
![Screenshot 2024-09-04 at 12 22 18 AM](https://github.com/user-attachments/assets/bed995d1-bbc5-4976-8e38-91ae9d17a5c2)
![Screenshot 2024-09-04 at 4 58 27 AM](https://github.com/user-attachments/assets/9f352d0a-bd16-4ae4-bde1-56bc9e654e0d)
![Screenshot 2024-09-04 at 4 55 58 AM](https://github.com/user-attachments/assets/4bd69209-33c4-4139-932c-e773ba7ec30d)

**Comment by meganrogge** [comment]
I am not following. When I enable a screen reader and focus each status bar item, I hear logical content only, including for the `{}` and `Typescript` or `JavaScript` sections.
